src/clients/ntp-client_auto.rb

```diff
@@ -76,9 +76,15 @@ def main
         #               cloning the just installed system
         if Mode.config && Stage.initial
           ntp_conf = "/etc/ntp.conf"
+          installed_ntp_conf = File.join(Installation.destdir, ntp_conf)
+          # if ntp is not installed at all we cannot copy it, so return empty values
+          if !::FileUtils.exist?(installed_ntp_conf)
```

Comment:
For reference, it it is a mistake in the original fix #30, later fixed in https://github.com/yast/yast-ntp-client/commit/f6e154482df4b1db66022cfe821c5a7f43e867d2.